libs/blocks/merch/merch.js

```diff
@@ -75,6 +75,7 @@ export async function getPriceContext(el, params) {
   const displayPerUnit = params.get('seat');
   const displayRecurrence = params.get('term');
   const displayTax = params.get('tax');
+  const forceTaxExclusive = params.get('forceTaxExclusive');
```

Comment:
Can we continue using shortened names for params in OST URLs, like `old`, `seat`, `term`, and `tax`?
Maybe `excl`?

Reply:
`exclusive` is fine as well

Reply:
or 'notax'?

Reply:
should i create a poll 😁